packages/calcite-components/src/components/table/table.stories.ts

```diff
@@ -773,6 +774,74 @@ export const selectionModeMultipleAndSelectedOnLoad_TestOnly = (): string =>
     </calcite-table-row>
   </calcite-table>`;
 
+export const selectionModeMultipleAndSelectedSelectionDisplayNoneOnLoad_TestOnly = (): string =>
```

Comment:
New stories can omit the `_TestOnly` suffix (see [updated conventions](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/conventions/Testing.md#screenshot-stories)).